### ISSUE TITLE
Problem with "url_launcher" package

### ISSUE THREAD

**Issue by edwinmendoza1999** [flutter/flutter#59732]
I have installed the package "url_launcher", but wanting to "flutter run" I have this error in the console "url_launcher_web not found. Please update settings.gradle".

They recommended me to modify the file "flutter app -> android -> settings.gradle". I did "flutter run" and it worked, but when I wanted to "flutter run build --apk" the console says "this is taking an unexpectedly long time" and finally throws errors.

**Comment by igorakkerman** [comment]
This might get fixed by #59294 once it is merged.

**Comment by MrBirb** [comment]
Well the linked PR was merged. Did it fix your issue @edwinmendoza1999 ?

**Comment by igorakkerman** [comment]
@MrBirb it was merged and later reverted after causing issues with other subprojects. This seems to have happened multiple times. I lost track of it. Anyone knows the latest state?

**Comment by darshankawar** [comment]
@edwinmendoza1999 
Can you confirm if you are still facing this issue using latest Flutter version ?

**Comment by pedromassangocode** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
